sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,102 +85,97 @@ client = FormRecognizerClient(endpoint, credential)
+based on inputs from a URL and inputs from a stream.
 
-### CustomFormClient
-A `CustomFormClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models. 
-It provides operations for training models on forms you provide, and extracting field values and locations from your 
-custom forms.  It also provides operations for viewing and deleting models, as well as understanding how close you 
-are to reaching subscription limits for the number of models you can train.
+### FormTrainingClient
+A `FormTrainingClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models.
+It provides operations for training models on forms you provide and operations for viewing and deleting models, as well as
+understanding how close you are to reaching subscription limits for the number of models you can train.
 
 ### Long-Running Operations
 Long-running operations are operations which consist of an initial request sent to the service to start an operation,
-followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has 
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
 succeeded, to get the result.
 
-Methods that train models or extract values from forms are modeled as long-running operations.  The client exposes 
-a `begin_<method-name>` method that returns an `LROPoller`.  Callers should wait for the operation to complete by 
-calling `result()` on the operation returned from the `begin_<method-name>` method.  Sample code snippets are provided 
+Methods that train models or recognize values from forms are modeled as long-running operations.  The client exposes
+a `begin_<method-name>` method that returns an `LROPoller`.  Callers should wait for the operation to complete by
+calling `result()` on the operation returned from the `begin_<method-name>` method.  Sample code snippets are provided
 to illustrate using long-running operations [below](#Examples).
 
 ### Training models
-Using the `CustomFormClient`, you can train a machine-learned model on your own form type.  The resulting model will 
-be able to extract values from the types of forms it was trained on.
+Using the `FormTrainingClient`, you can train a machine-learned model on your own form type.  The resulting model will
+be able to recognize values from the types of forms it was trained on.
 
 #### Training without labels
-A model trained without labels uses unsupervised learning to understand the layout and relationships between field 
-names and values in your forms. The learning algorithm clusters the training forms by type and learns what fields and 
-tables are present in each form type. 
+A model trained without labels uses unsupervised learning to understand the layout and relationships between field
```

Comment:
Should we call it out as unsupervised or unlabeled here?

Reply:
to be consistent with service documentation, we should say unlabeled

Reply:
ok. I didn't change these because I've never heard of unlabeled learning, just unsupervised learning. But I definitely understand where you're coming from with making it consistent with the service. I'll bring it up during scrum. Thanks!

Reply:
I would just remove the part about unsupervised learning. "A model trained without labels understands the layout..." or say something broad like "machine learning".